src/constructors.jl

```diff
@@ -179,3 +182,11 @@ julia> TaylorN(Rational{Int},2)
 TaylorN{T<:Number}(::Type{T}, nv::Int; order::Int=get_order()) =
     return TaylorN( [HomogeneousPolynomial(T, nv)], order )
 TaylorN(nv::Int; order::Int=get_order()) = TaylorN(Float64, nv, order=order)
+
+
+# A `Number` which is not an `AbstractSeries`
+const NumberNotSeries = Union{setdiff(subtypes(Number), [AbstractSeries])...}
```

Comment:
Somehow beyond my understanding, I think this is at the core why problems reported in #62 and #94 are solved.... or Julia-magic.